src/client/thirdParty/Inkeep.tsx

```diff
@@ -84,7 +103,16 @@ const useInkeepSettings = (): InkeepSharedSettings => {
 };
 
 export function InkeepChatButton() {
-  const { baseSettings, aiChatSettings, searchSettings, modalSettings } = useInkeepSettings();
+  const { data } = useVisitorData({ extendedResult: true, timeout: FPJS_CLIENT_TIMEOUT });
+  const visitorId = data?.visitorId;
+
+  const settings = useInkeepSettings(visitorId ?? '');
+
+  if (!visitorId || !settings) {
+    return null;
+  }
+
+  const { baseSettings, aiChatSettings, searchSettings, modalSettings } = settings;
```

Comment:
Why deconstruct `settings` only to reconstruct them the same as `chatButtonProps`? Why not just 

```tsx
export function InkeepChatButton() {
  const settings = useInkeepSettings();
  return <ChatButton {...settings} />;
}
```